src/nspawn/nspawn.c

```diff
@@ -5798,7 +5798,7 @@ static int run(int argc, char *argv[]) {
                 arg_quiet = true;
 
         if (!arg_quiet)
-                log_info("Spawning container %s on %s.\nPress ^] three times within 1s to kill container.",
+                log_info("Spawning container %s on %s.\nPress ctrl-] three times within 1s to kill container.",
```

Comment:
Maybe `Ctrl` (capitalized) is better? Though, not a strong opinion.

Reply:
Agreed but equally not fuzzed.

Reply:
Okay, pushed that change